Packs/GreyNoise/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,11 @@
+#### Integrations
+##### GreyNoise
+- Fixed bug with "test configuration" validation always failing
+- Updated "ip" command to perform both NOISE and RIOT lookups
+- Added greynoise-context command to only perform NOISE lookup
+- Updated "ip-quick-check" command to include RIOT boolean
+- Updated URL Hyperlink URLs to new Visualizer base URL
+- Update IP Context Table to include additional elements
+- Added "cve" lookup option to "query" command
+##### GreyNoise Community
+- None
```

Comment:
I see that you did make changes to GreyNoise community, didn't you?

Reply:
In which case, let's write `Maintenance and stability enhancements.` instead

Reply:
Sure, no problem.